src/main/java/com/diablo3CharViewer/data_models/HeroDataModel.java

```diff
@@ -43,6 +43,38 @@ public HeroDataModel(int id, String name, String classHero, int level, int parag
         this.stats = stats;
     }
 
+    public int getId() {
```

Comment:
Wild thought: can you make all those fields in the class final? I don't really see a need for them to be mutable ATM

Reply:
done